chromium_src/chrome/browser/component_updater/registration.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../chrome/browser/component_updater/registration.h"
```

Comment:
just #include "chrome/browser/component_updater/registration.h"?

Reply:
Fixed in 00a17f6f17500a72e33cd0760ab2e297b9db06ec